### ISSUE TITLE
(Request) Ability to clear cached downloads from pkgi

### ISSUE THREAD

**Issue by fennectech** [mmozeiko/pkgi#22]
I would find the ability to remove partal downloads from within pkgi useful. Thanks for pkgi.

**Comment by mmozeiko** [comment]
I'm thinking something like this:

* if you click [X] on item that has no resume data, it will download & install it from scratch
* if you click [O] on item that has resume data, it will ask to delete it, and if you confirm - then deletion will happen
* if you click [X] on item that has resume data, it will ask "do you want to resume or start download from beginning", then you can choose - either to resume download, or delete it first and then start download from scratch (or third option - cancel this selection).

Does this sounds reasonable?

**Comment by ghost** [comment]
Good idea, it's reasonable.

**Comment by fennectech** [comment]
Sounds perfect   Would also like an option to delete ALL resume data at once.

**Comment by opoisso893** [comment]
good idea to be able to select "resume" or "start download" or "delete resume data"

**Comment by wsy741963** [comment]
Can download some free games witch i'd never download from the PSN before?

**Comment by MarioMasta64** [comment]
perhaps [ ] for options? you could bring up a menu something like: more details (which would bring up your fancy gui) and delete package data :D and maybe more

**Comment by mmozeiko** [comment]
what is `[ ]` ? space bar? where do you get it on vita?

**Comment by MarioMasta64** [comment]
square

**Comment by mmozeiko** [comment]
doh :) I'm thinking X will bring up details, with default button selected to install (when X is pressed again), or optionally you can choose DLCs/patch. I want to leave square for marking multiple items from list - similar how in VitaShell you can mark multiple files to delete/move them at the same time.

**Comment by MarioMasta64** [comment]
ooo. multi-install soon? :^)

**Comment by fennectech** [comment]
Usually triangle is used for those details things.

**Comment by rafaelqueiroz89** [comment]
so, is there a way to remove cached files that were not downloaded? There is an empty circle whenever i try to download and it doesn't finish it, how can i clear this cache?
